formulaire_modification.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="fr">
+    <head>
+        <!-- Ici se trouvent les métadonnées pour le navigateur -->
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        
+
+        <!-- Ici se trouvent les métadonnées pour le moteur de recherche (et le navigateur) -->
+        <meta
+            name="description"
+            content="Ce petit paragraphe sera affiché dans les SERP, juste sous le titre des pages."
+        />
+
+        <!-- Ici se trouvent les liens vers les fichiers favicons -->
+        <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
+        <link rel="manifest" href="site.webmanifest">
+        <meta name="msapplication-TileColor" content="#da532c">
+        <meta name="theme-color" content="#ffffff">
+
+        <!-- Ici se trouvent les liens vers les feuilles de styles -->
+        <link rel="stylesheet" href="assets/css/styles.min.css" />
+    </head>
+
+    <body>
+        <div class="row">
+            <section class="block md color-sky">
+                <form action="">
+                    <h2>Modifier vos données</h2>
+                    <h4 class="form-field"><u>Modifier votre adresse mail</u></h4>
+                    <div class="form-field">
+                        <label for="firstName">Ancienne adresse mail :*</label>
```

Comment:
Je crois que j'ai oublié ce détail aussi sur ma PR, je vais aller vérifier ensuite, mais il nous faudrait modifier le type, l'id, et le name des labels et input afin qu'il soit plus représentatif des champs demandés.

label : for="oldMail"